### PR TITLE
correct typo in datacard

### PR DESCRIPTION
**Pull request by Rensvandeschoot**

### CODE REVIEW

File: asreview/webapp/src/ProjectComponents/SetupComponents/DataComponents/DataForm.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ const DataForm = (props) => {
               projectHasDataset={props.info?.projectHasDataset}
               primaryDefault="Add prior knowledge"
               primaryAdded="Prior knowledge added"
-              secondaryDefault="Indicate your preference with at least 1 relevant and 1 irrelevant records"
+              secondaryDefault="Indicate your preference with at least one relevant and one irrelevant record"
```
Comment (terrymyc) [inline asreview/webapp/src/ProjectComponents/SetupComponents/DataComponents/DataForm.js:132]:
```suggestion
              secondaryDefault="Indicate your preference with at least 1 relevant and 1 irrelevant record"
```

Comment (terrymyc) [reply asreview/webapp/src/ProjectComponents/SetupComponents/DataComponents/DataForm.js:132]: Check https://www.material.io/design/communication/writing.html#principles
> Use numerals ( "1, 2, 3," not "one, two, three")

**Review by terrymyc — APPROVED** [review]

**Comment by Rensvandeschoot** [comment]
Which style guide do we use for numerals? For scientific papers I am used to spell out numbers up to a value of 10.

**Comment by terrymyc** [comment]
Check Material Design: https://www.material.io/design/communication/writing.html#principles

**Pull request merged**